bundle/src/main/java/com/adobe/acs/commons/indesign/dynamicdeckdynamo/servlets/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright 2015 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+@Version("4.4.3")
```

Comment:
Likely can be 1.0.0

Reply:
Also, does this need to be in a public package? I'd think it would be in an impl somewhere